css/index.css

```diff
@@ -0,0 +1,247 @@
+html {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  body {
+    background: #f8f8f8;
+    color: #303438;
+    font-family: "Lato","Helvetica Neue",Helvetica,Arial,sans-serif;
+    margin: 0;
+    padding: 0;
+  }
+  svg {
+    fill: currentColor;
+  }
+  
+  a {
+    color: #1F0C7B;
+    text-decoration: none;
+  }
+    a:hover {
+      text-decoration: underline;
+      cursor: pointer;
+    }
+  
+  h1 {
+    text-align: center;
+    margin-bottom: 1.5em;
+    font-size: 2.5em;
+    font-family: "Merriweather",serif;
+  }
+  div + h1 {
+    margin-top: 3em;
+  }
+  @media screen and (max-width: 600px) {
```

Comment:
[pebble] I think for consistency we should use the [USWDS responsive screen breakpoints](https://designsystem.digital.gov/utilities/layout-grid/#responsive-variants-2). It sets tablet as >= 640px.

Reply:
Same for other media queries in this file.